assets/scss/modules/_forms.scss

```diff
@@ -0,0 +1,391 @@
+form {
+  text-align: center;
+  font-family: $dp-font-family-proximanova;
+
+  .rgpd {
```

Comment:
what this means?

Reply:
the classes created should be more semantic: "policy - rgpd" or "privacy - rgpd"  :)

Reply:
Also, we use english in our styles sheets, so it should be: General Data Protection Regulation (GDPR)